### ISSUE TITLE
Added to group but I don't know how to start it.

### ISSUE THREAD

**Issue by TazaDeCafee** [deptyped/notepher-bot#37]
I love this bot, but when I tried to use it in a group, I have added it but I don't know how to start, I tried with /start /new /list, list, new, add, start, I have tried adding @notepherbot to all of these commands, but no one worked. 😭

**Comment by deptyped** [comment]
The bot does not support groups. What is your use case? Do you want to take notes together with other group members?

**Comment by TazaDeCafee** [comment]
Yes, I want to take notes together, I thought it could be used in group because there's an option to add to groups. Others note/task/todolist that doesn't work for group doesn't have that option
![Screenshot_20240217-000040](https://github.com/deptyped/notepher-bot/assets/160249806/12b1ba1d-942a-47fc-9099-ab19c437b7d7)

**Comment by Zequez** [comment]
Oh man, I feel sad to discover that it cannot be added to groups, it was the exact use case I wanted. I want to manage a list of notes as a group within Telegram groups; that would be a killer feature for me.
